includes/class-amp-travel-cpt.php

```diff
@@ -196,13 +205,31 @@ public function adventure_meta_box_html() {
 	 */
 	public function save_adventure_post() {
 
-		// First check if the amp_travel_price exists at all -- it doesn't in Gutenberg.
-		if ( isset( $_POST['amp_travel_price'] ) && ! wp_verify_nonce( $_POST['amp_travel_price_nonce'], basename( __FILE__ ) ) ) {
-			return;
-		}
+		// This check is needed since otherwise Gutenberg save will fail -- it uses different saving logic.
+		if (
+			isset( $_POST['amp_travel_start_date'] )
+			||
+			isset( $_POST['amp_travel_end_date'] )
+			||
+			isset( $_POST['amp_travel_price'] )
+		) {
+			global $post;
+
+			if ( ! wp_verify_nonce( $_POST['amp_travel_adventure_nonce'], basename( __FILE__ ) ) ) {
+				return;
+			}
+
+			if ( isset( $_POST['amp_travel_price'] ) ) {
+				update_post_meta( get_the_ID(), 'amp_travel_price', esc_attr( $_POST['amp_travel_price'] ) );
```

Comment:
Hi @miina, 
It looks like you were just following the example of the code from `develop`, which used `esc_attr()` to sanitize `$_POST`. I should have mentioned this earlier.

But could you please sanitize the `$_POST` value like:
```php
sanitize_text_field( wp_unslash( $_POST['amp_travel_price'] ) )
```

Could you please do the same for these values below:
* `$_POST['amp_travel_start_date']`
* `$_POST['amp_travel_end_date']`